File: app/src/test/java/com/brewery/training/playground/HelloTest.java
package com.brewery.training.playground;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class HelloTest {
    @Test
    public void testGreetWithAName() {
        assertEquals("Hello, Dolly!", Utilities.greeting("Dolly"));
    }

    @Test
    public void testGreetWithoutAName() {
        assertEquals("Hello, World!", Utilities.greeting());
    }
}

File: app/src/test/java/com/brewery/training/playground/demo/HelloTest.kt
package com.brewery.training.playground.demo

import org.junit.Test

import org.junit.Assert.*

class HelloTest {

    @Test
    fun `check if strings are palindromes`() {
        assertTrue("Madam, in Eden, I'm Adam".isPalindrome())
        assertTrue("Flee to me, remote elf!".isPalindrome())
    }

    @Test
    fun `verify that the method doesn't just return true`() {
        assertFalse("This is not a palindrome".isPalindrome())
    }
}